### PR TITLE
Make tasks core only a compatibility layer to proxy plugin

### PR DESCRIPTION
**Pull request by adamruzicka**
requires:
- [ ] https://github.com/theforeman/smart_proxy_dynflow/pull/81

### CODE REVIEW

**Comment by ezr-ondrej** [comment]
What about the `ContinuousOutput` class?

File: lib/foreman_tasks_core/continuous_output.rb [hunk 1]
```diff
@@ -1,50 +0,0 @@
-module ForemanTasksCore
-  class ContinuousOutput
```
Comment (ezr-ondrej) [inline lib/foreman_tasks_core/continuous_output.rb:2]: Didn't we want to keep this?
Maybe transfering to a `ForemanTasks` namespace? :thinking:

**Review by ezr-ondrej — COMMENTED** [review]
should we alias the actions ?
`OutputCollectorAction` and so on just in case we have some dynflow actions in the database still not finished, they would start failing immediatelly, right?

**Comment by adamruzicka** [comment]
By default dynflow on the smart proxy uses an in memory database so we should start with a clean slate after each process restart. It could be an issue if someone had a persistent db though :thinking:

**Comment by ezr-ondrej** [comment]
> It could be an issue if someone had a persistent db though

How often that happens, do we support such cases and how hard it is to recover from such situation?
I can't judge here but if it can give someone hard time, I'd prefer to have the aliases just in case

**Comment by adamruzicka** [comment]
> How often that happens

No idea, but on-disk sqlite is borderline unusable with our access pattern.

> how hard it is to recover from such situation?

Remove the db and start over? Even if there was no issue with action classes, I'm not sure if any of the actions running on the proxy is actually resumable

**Review by ezr-ondrej — APPROVED** [review]
Works well, let's get this in :+1:

**Pull request merged**